IFC 4.3/UT_Alignment_12d_11/UT_Alignment_12d_11.ifc

```diff
@@ -1,49 +1,52 @@
-#35 = IFCALIGNMENTVERTICAL('3gA2mMqFX0xfZWiXfZxgI6', #2, 'Vertical Alignment', '12d Vertical Alignment', $, $, $, (#36, #37, #38, #39, #40));
-#36 = IFCALIGNMENTVERTICALSEGMENT($, $, 25., 22.1775718912935, 500.2564, 8.07402171512695E-2, 8.07402171512695E-2, $, .CONSTANTGRADIENT.);
-#37 = IFCALIGNMENTVERTICALSEGMENT($, $, 47.1775718912935, 20., 502.047021970391, 8.07402171512308E-2, -9.12280701755666E-2, 116.300512791602, .PARABOLICARC.);
-#38 = IFCALIGNMENTVERTICALSEGMENT($, $, 67.1775718912935, 8.0492098531853, 501.942143440148, -9.12280701752541E-2, -9.12280701752541E-2, $, .CONSTANTGRADIENT.);
-#39 = IFCALIGNMENTVERTICALSEGMENT($, $, 75.2267817444788, 30., 501.207829558806, -9.12280701754341E-2, 4.4056649648193E-2, 221.754533986629, .PARABOLICARC.);
-#40 = IFCALIGNMENTVERTICALSEGMENT($, $, 105.226781744479, 19.7732182555212, 500.500258250897, 4.40566496482954E-2, 4.40566496482954E-2, $, .CONSTANTGRADIENT.);
+#1 = IFCPROJECT('2LJesmETv6W9jWXnI$q1Aj', #2, 'UT Alignment 12d - 11', 'Alignment with VG shorter than HG', $, $, $, (#13), #8);
+#2 = IFCOWNERHISTORY(#3, #7, $, .NOCHANGE., $, $, $, 1612674395);
+#3 = IFCPERSONANDORGANIZATION(#4, #5, $);
+#4 = IFCPERSON($, 'Gregory', 'Lee', $, $, $, $, $);
+#5 = IFCORGANIZATION($, '12d Solutions - Ranch', $, (#6), $);
+#6 = IFCACTORROLE(.CONSULTANT., 'Engineering', $);
+#7 = IFCAPPLICATION(#5, '15.0 Alpha 480 [DEVELOP] Themes are Enabled MK 5', '12d Model', '12d Model');
+#8 = IFCUNITASSIGNMENT((#9, #10, #11, #12));
+#9 = IFCSIUNIT(*, .LENGTHUNIT., $, .METRE.);
+#10 = IFCSIUNIT(*, .AREAUNIT., $, .SQUARE_METRE.);
+#11 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#12 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#13 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #15, #10019);
+#14 = IFCLOCALPLACEMENT($, #15);
+#15 = IFCAXIS2PLACEMENT3D(#16, #17, #18);
+#16 = IFCCARTESIANPOINT((0., 0., 0.));
+#17 = IFCDIRECTION((0., 0., 1.));
+#18 = IFCDIRECTION((1., 0., 0.));
+#19 = IFCDIRECTION((0., 1., 0.));
+#10019 = IFCDIRECTION((0., 1.));
+#20 = IFCSITE('3FE55cJIj1EBn9XOYtHi8t', #2, 'Default Site', 'Description of Default Site', $, #14, $, $, .ELEMENT., $, $, $, $, $);
+#21 = IFCRELAGGREGATES('3RGWsbicz9GAGSuC1FRNdm', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#20));
+#22 = IFCFACILITY('25b4Qh$85Dm9ItvVaRrbdM', #2, 'SA HG partial clothoid VG short', 'Description of SA HG partial clothoid VG short', $, #14, $, $, .ELEMENT.);
+#23 = IFCRELAGGREGATES('2seZ2zVjT9NOhJaip4imZX', #2, 'SiteContainer', 'SiteContainer For All Models', #20, (#22));
+#24 = IFCRELCONTAINEDINSPATIALSTRUCTURE('31h6tWQ4XET8jVBSwfNZDw', #2, 'FacilityContainer', 'FacilityContainer for Elements', (#25), #22);
+#25 = IFCALIGNMENT('0y2hQsQKfAqxEufY7Z_1jL', $, 'RS02 short VG', '12d Alignment String', 'natural clothoid', #14, $, #10025, $);
+#10025 = IFCPOLYLINE((#10026,#10027));
+#10026 = IFCCARTESIANPOINT((0., 0., 0.));
+#10027 = IFCCARTESIANPOINT((0., 1., 0.));
```

Comment:
This needs attention by @SergejMuhic . We need to discuss the faith of the `IfcAlignment.Axis`.